edgedb/blocking_client.py

```diff
@@ -97,6 +103,17 @@ async def connect_addr(self, addr, timeout):
 
             self._protocol = proto
             self._addr = addr
+            self._ping_wait_time = max(
+                (
+                    getattr(
+                        self.get_settings().get("system_config"),
+                        "session_idle_timeout",
+                        DEFAULT_SESSION_IDLE_TIMEOUT,
```

Comment:
I don't remember -- can it be missing or we always send it? (I think we always do, no?)

Reply:
Yeah, I also think we always send:

https://github.com/edgedb/edgedb/blob/60c07267b310bffbf9a827635b1d51be4a71d72a/edb/server/bootstrap.py#L1131-L1141

then

https://github.com/edgedb/edgedb/blob/60c07267b310bffbf9a827635b1d51be4a71d72a/edb/server/protocol/binary.pyx#L536-L539

Reply:
Then there's no need for DEFAULT_SESSION_IDLE_TIMEOUT?

Reply:
Right!